src/snowflake/cli/plugins/snowpark/package/anaconda.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, packages):
     def is_package_available(
         self, package: Requirement, skip_version_check: bool = False
     ) -> bool:
+        if package.name is None:
```

Comment:
```suggestion
        if not package.name:
```
May help with handling empty strings also